akka-docs/src/main/paradox/typed/fault-tolerance.md

```diff
@@ -38,3 +45,20 @@ Java
 For a full list of strategies see the public methods on `SupervisorStrategy`
 
 
+## Bubbling exceptions up
+
+In some scenarios it may be useful to push the decision about what to do on a failure upwards in the Actor hierarchy
+(in untyped Akka Actors this is how it works by default) and let the parent actor handle what should happen on failures.
+
+// FIXME is it useful that we should introduce something for it perhaps?
+
+// IDEA 1: parent will have to use watch, and act on children terminating
+// problems: no idea what exception happened (maybe that is good, to not tightly couple parent and children)
+
+// IDEA 2: provide a special supervision strategy that queries the parent, 
+// put's the actor in limbo state until the parent has decided
+// problems: maybe this is just complicating things and not actually so useful? 
```

Comment:
that would pretty much mean that we would be back to that the supervision logic would run in the parent actor, with all the complicated stuff for implementing that if/when we create a new runtime

Reply:
Yeah, that's what I was thinking as well.